### PR TITLE
Add co-occurring WikiProject panel

### PR DESCRIPTION
**Pull request by carlinmack**
Daniel mentioned that it would be useful to see co-occurring WikiProjects like we do with co-occurring topics on the topics aspect

#### WikiProject Invasion Biology    
image

#### WikiProject Covid-19
image

#### WikiProject Film
image

### Caveats

> Please list anything which has been left out of this PR or which should be considered before this PR is accepted
Check any of the following which apply:

* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ]  This change requires a documentation update
    * [ ]  I have made corresponding changes to the documentation
* [ ]  This change requires new dependencies (please list)

*If you make changes to the Python code*
  
* [ ]  My code passes the [tox](https://tox.readthedocs.io/en/latest/) check, I can receive warnings about tests, documentation or both

### Testing
Please describe the [tests](https://numpy.org/doc/stable/reference/testing.html) that you ran to verify your changes. Provide instructions, so we can reproduce. Please also list any relevant details for your test configuration.

* Test A
* Test B

### Checklist
* [ ] I have [commented](https://peps.python.org/pep-0008/#comments) my code, particularly in hard-to-understand areas
* [ ] My changes generate no new warnings
* [ ] I have not used code from external sources without attribution
* [ ] I have considered [accessibility](https://www.w3.org/standards/webdesign/accessibility) in my implementation 
* [ ] There are no remaining debug statements (print, console.log, ...)

### CODE REVIEW

File: scholia/app/templates/wikiproject.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 {{ sparql_to_table("maintained") }}
 {{ sparql_to_table("focus") }}
+{{ sparql_to_table('co-occurring') }}
```
Comment (fnielsen) [inline scholia/app/templates/wikiproject.html:9]: This id does not correspond to the id in the HTML tag.

**Comment by fnielsen** [comment]
https://scholia.toolforge.org/wikiproject/Q115206846#co-occurring-wikiprojects

**Comment by fnielsen** [comment]
I changed the filename back to what you originally had. Would this not be the most "natural"?

**Comment by carlinmack** [comment]
I was matching https://github.com/WDscholia/scholia/blob/master/scholia/app/templates/topic_co-occurring.sparql

**Comment by fnielsen** [comment]
Ok. I see. I do not know whether we should change it then. Currently it fits with the headline, but there is inconsistency.

**Pull request merged**